scripts/scripts.js

```diff
@@ -122,6 +169,7 @@ function decorateBlocks($main) {
  * @param {string} blockName name of the block
  * @param {any} content two dimensional array or string or object of content
  */
+// eslint-disable-next-line no-unused-vars
```

Comment:
Really needed ?

Reply:
This function is no longer used by default, but could theoretically be used to create synthetic/auto blocks. But for that purpose, it would need to be exported...

Reply:
i think this is a function that i think we should encourage to use, possibly supported with an example. i think one of the most typical examples would be a hero/marquee section that is turned into a hero block. i have come to believe that somehow the templates or page types have a close connection with autoblocking.
meaning that if you have eg. a `category` page or a `article` page or similar those would create different auto blocks from the default content. somehow feels like an intuitive concept but i am not sure how to marry the two...
i think for now we could possibly just use an `h1` with a preceding `picture` and turn that into a `marquee` block as an example.